### ISSUE TITLE
Training config: Allow for more settings to be changed from config file

### ISSUE THREAD

**Issue by DanielNobbe** [DanielNobbe/Forex#31]
Add to config file handling:

- [ ] optimizer
- [ ] loss function
- [ ] learning rate
- [ ] others..?
